### DISCUSSION TITLE
Welcome to inequality Discussions!

### DISCUSSION THREAD

**Discussion by jGaboardi** [pysal/inequality#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OMFAC** [comment]
hola buenas noches, en este momento estoy intentando instalar el modulo de INEQUALITY en ARCGIS PRO, pero no lo he logrado, quiero realizar un analisis con el coeficiente de GINNI con unos datos prediales. me podrias ayudar. muchas gracias
